R/utils_render_table_non_registers.R

```diff
@@ -1,37 +1,53 @@
+render_non_register_tables_html <- function(list_reg_tables, page_type){
+  if (page_type == "codecheckers") {
```

Comment:
Could the check also be "if `list_reg_tables` is of type list, then return it, otherwise wrap it in a list" ?

That could make it more independent from the page type (yet another constant hidden in the code).

`is.list()` could do that.

Reply:
Resolved!